features/step_definitions/sample_file_steps.rb

```diff
@@ -0,0 +1,219 @@
+Given(/^a demo directory with a smelly file$/) do
+  contents = <<-EOS
+class Dirty
```

Comment:
Yes, but neatly abstracted away in an easy to integrate, well tested library..;)